library/channel.c

```diff
@@ -120,13 +120,19 @@ struct msg_receiver {
 };
 
 static void ch_init_stream(ziti_channel_t *ch) {
-    if (ch->connection == NULL) {
-        ch->connection = calloc(1, sizeof(*ch->connection));
-        tlsuv_stream_init(ch->loop, ch->connection, ch->ctx->tlsCtx);
-        tlsuv_stream_keepalive(ch->connection, true, 30);
-        tlsuv_stream_nodelay(ch->connection, true);
-        ch->connection->data = ch;
+
+    if (ch->connection != NULL) {
```

Comment:
ch->connection will be leaked at line 130

Reply:
this is a leftover from debugging. this function should not be called when `ch->connection != NULL`